resolver/terraform.go

```diff
@@ -23,6 +23,11 @@ func NewTerraformResolver(services []*terraform.FastlyService) []Resolver {
 			ServiceName: v.Name,
 		}
 		for _, vcl := range v.Vcls {
+			// Always save module names with .vcl extension
+			if !strings.HasSuffix(vcl.Name, ".vcl") {
+				vcl.Name = fmt.Sprintf("%s.vcl", vcl.Name)
+			}
```

Comment:
@ysugimoto I think we should always assume the statement `include "filename.vcl"` should has an extension no matter what, thus we should always insert the module names with extensions. what do you think?

Reply:
My concern is that trims `.vcl` extension on resolving module.
By removing the extension but even `stmt.Module.Value` may have `some_module.vcl` then could we solve it?

I means that a custom vcl is written something like `include "some_module";` I'm guessing it may not solve them so I just want to regard to do it by adding test cases.

Reply:
ahh so you think it is ok to have `include "some_module"` and not `include "some_module.vcl"`
I thought the include statement should be followed by "module_name + vcl" always

Reply:
If you think we should allow this case, I can add that!

Reply:
Yes I think we should allow that case because my project defines custom vcl without extension.
For example,

```vcl
// main.vcl
include "some_module";
```

```hcl
// main.tf
resource "fastly_service_vcl" "..." {
... 
  vcl {
    name = "some_module"
    content = "..."
  }
...
```

So I'm happy to allow the above case for backward compatibility!